### DISCUSSION TITLE
Clean Todo App with Riverpod

### DISCUSSION THREAD

**Discussion by titomitto** [rrousselGit/riverpod#2382] [Show and tell]
Hey devs! I recently created a Todo app that uses Clean Architecture and Riverpod. You can check out the code here: https://github.com/TitoMitto/clean-todo. I would love to get your feedback and suggestions on how to improve it. The app is built with a focus on separation of concerns, testability, and maintainability, making it easy to understand and modify. So, feel free to take a look and let me know what you think!

**Comment by temcewen** [comment]
Hey I really like it! It's very clean and organized.

**Reply by titomitto** [reply]
Thanks
